clusters/build-clusters/01_cluster/README.md

```diff
@@ -120,6 +120,35 @@ It is automated by config-updater:
 * alertmanager secret for sending notification to slack
 * PVCs for monitoring stack
 
+## Upgrade the cluster
+
+### Updating between minor versions
+
+Upgrade channel configuration, e.g., from OCP 4.3 to 4.4
+
+```
+oc --as system:admin --context build01 patch clusterversion version -p '{"spec":{"channel":"stable-4.4"}}'
+```
+
+### Run the upgrade command
+
+Choose on the [release page](https://openshift-release.svc.ci.openshift.org/) the version to upgrade to, for example
+[4.4.0-rc.12](https://openshift-release.svc.ci.openshift.org/releasestream/4-stable/release/4.4.0-rc.12):
+
+```
+$ bash clusters/build-clusters/01_cluster/hack/upgrade-cluster.sh quay.io/openshift-release-dev/ocp-release:4.4.0-rc.12-x86_64
+--context build01 adm upgrade --allow-explicit-upgrade --to-image quay.io/openshift-release-dev/ocp-release@sha256:674200aaa1cc940782aa4b109ebfeaf7206bb57c3e01d62ec8e0b3d5ca910e8f
+```
+
+Double-check the SHA with:
+
+```
+$ curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.4.0-rc.12/release.txt | grep Pull
```

Comment:
nit: This is skipping signature verification, exposing you to attacks via a compromised mirror.  Although the CVO will check for an official Red Hat signature on the image itself, so as long as you don't force past that check, the worst an attacker could do would be to feed you a different release's digest (e.g. they could feed you the 4.0.0 release digest, which would probably brick your cluster, but at least it wouldn't install an exploit that leaked creds).

Reply:
This is a manual `double-check` anyway.
The value i use in upgrade is from the oc-release-info command.
Unless both commands are hijacked, we should be good.

No `--force` any more in the future unless confirmed by installer experts.

Reply:
"You need to compromise both the registry and the mirrors" is better than "compromising just the mirrors is enough".  But still, checking the signature and hashes on the mirrors isn't *that* bad.  Depends on your risk tolerance, I guess...